packages/cli/src/eject/index.ts

```diff
@@ -0,0 +1,39 @@
+import fs from 'node:fs'
+
+import chokidar from 'chokidar'
+
+import { IGNORED_PATH, WATCH_PATH } from '../config/constants'
+
+export const ejectAction = async () => {
+  const watcher = chokidar.watch(WATCH_PATH, {
+    ignored: IGNORED_PATH,
+  })
+
+  const importScripts: string[] = []
+  const modelObject: string[] = []
+  let types = 'export type Model ='
+
+  watcher.on('add', async (modelPath) => {
+    const modelName = modelPath.split('/').slice(1, -1).join(':')
```

Comment:
so, its possible to preserve the slash in the real models using nested objects.

if you have a models folder which contains

```
foo.tsx
bar/baz.tsx
```

your model definition file would be:

```tsx

const models = {
  foo,
  bar: { baz }
}
```

so it would be great to support that at some point

Reply:
It's great idea